TEAMS.md

```diff
@@ -26,13 +26,21 @@
 
 [@ekcasey][@ekcasey], [@jromero][@jromero], [@sclevine][@sclevine]
 
+## Distribution Team
+
+> The Distribution team is responsible for maintaining tools and services that support the distribution, discovery, and integration of buildpacks.
+
+### Maintainers
+
+TBD
+
```

Comment:
Move below ### Project Contributors

Reply:
Fixed in 68f7b86

Reply:
It still looks like all of the Platform team Contributors are being deleted here.

Reply:
ah, i see what you mean. i think have it now with 4f434f4